gravity.sh

```diff
@@ -394,7 +395,7 @@ gravity_DownloadBlocklists() {
 
   if [[ "${status}" -ne 0 ]]; then
     echo -e "\\n  ${CROSS} Unable to create new database ${gravityTEMPfile}\\n  ${output}"
-    gravity_Cleanup "error"
+    #gravity_Cleanup "error"
```

Comment:
Why did you comment this out?

Reply:
Thanks for catching this. It is indeed a debugging leftover because I wanted to have access to the temp database after a failure to create the database. I will undo this.